src/symbols.js

```diff
@@ -684,27 +684,33 @@ defineSymbol(text, main, spacing, "\u00a0", "~");
 // There are lots of symbols which are the same, so we add them in afterwards.
 
 // All of these are textords in math mode
-const mathTextSymbols = "0123456789/@.\"";
-for (let i = 0; i < mathTextSymbols.length; i++) {
-    const ch = mathTextSymbols.charAt(i);
+for (const ch of "0123456789/@.\"") {
```

Comment:
Nice cleanup, but I think this may actually cause the bundle size to increase quite a bit b/c it requires a polyfill for the string iterator which I believe uses .codePointAt() which also ends up be polyfilled.  Can you generate a build and see how much the bundle size has increased?

Reply:
I don't think there are polyfills involved, but this is what it expands to:

```js
var _iteratorNormalCompletion = true;
var _didIteratorError = false;
var _iteratorError = undefined;

try {
    for (var _iterator = (0, _getIterator3.default)("0123456789/@.\""), _step; !(_iteratorNormalCompletion = (_step = _iterator.next()).done); _iteratorNormalCompletion = true) {
        var _ch = _step.value;

        defineSymbol(math, main, textord, _ch, _ch);
    }

    // All of these are textords in text mode
} catch (err) {
    _didIteratorError = true;
    _iteratorError = err;
} finally {
    try {
        if (!_iteratorNormalCompletion && _iterator.return) {
            _iterator.return();
        }
    } finally {
        if (_didIteratorError) {
            throw _iteratorError;
        }
    }
}
```

Or minified:

`var M=true;var _=false;var S=undefined;try{for(var z=(0,n.default)('0123456789/@."'),T;!(M=(T=z.next()).done);M=true){var A=T.value;u(o,f,k,A,A)}}catch(e){_=true;S=e}finally{try{if(!M&&z.return){z.return()}}finally{if(_){throw S}}}var C=true;var O=false;var N=undefined;try{for(var j=(0,n.default)('0123456789!@*()-=+[]<>|";:?/.,'),L;!(C=(L=j.next()).done);C=true){var E=L.value;u(s,f,k,E,E)}}catch(e){O=true;N=e}finally{try{if(!C&&j.return){j.return()}}finally{if(O){throw N}}}`

So it does add a few hundred bytes.  Shall I switch back?

Reply:
The code that it generates is pretty awful.  We should experiment with `loose` mode for the `for-of` plugin for babel sometime, see https://babeljs.io/docs/plugins/transform-es2015-for-of.  Interestingly, when iterating over an array literal it will output a basic `for` loop.  Maybe we could do `for (const letter of "abc...".split()) { ... }` to force this optimization.

Reply:
How about the following, which Babel seems to leave as-is?

```js
"0123456789/@.\"".split().forEach(function () {
    defineSymbol(math, main, textord, ch, ch);
});
```

Wow, this is actually fastest!  [jsperf benchmark](https://jsperf.com/iterate-over-string)

![jsperf output](https://user-images.githubusercontent.com/2218736/34009558-5e51b300-e0d7-11e7-9192-5d7dd1a60e8c.png)

Reply:
I wasn't expecting that.  Awesome!

Reply:
Oops, that's because I was using `split` wrong.  It's now the slowest...  So perhaps I should go back to a regular for loop.

![Corrected jsperf output](https://user-images.githubusercontent.com/2218736/34010243-f760e8de-e0d9-11e7-8915-93eb953c3ea0.png)

Reply:
right, b/c it still has to call `charCodeAt`... yeah, let's go back to regular for loops.